src/types/metaFactory.ts

```diff
@@ -117,5 +131,31 @@ export const MetaFactoryTypes: MetaFactoryTypes = {
           }
         }
       })}
+  },
+  candyMachine: {
+    newContractCreatedEvent: '',
+    metadataAccountToNftFactory: (metadataAccount, metaFactory) => {
+      return {
+        id: metadataAccount.mintAddress.toBase58(),
+        contractType: NFTContractTypeName.candyMachine,
+        platformId: metaFactory.platformId,
+        standard: NFTStandard.METAPLEX,
+        name: metadataAccount.name,
+        symbol: metadataAccount.symbol,
+        autoApprove: true, 
+        approved: true, 
+        typeMetadata: {
+          collection: metaFactory.id,
```

Comment:
instead of collection, lets go with something like:

<img width="812" alt="image" src="https://user-images.githubusercontent.com/743865/198274077-22afad97-4108-4911-8a42-2d11104e709d.png">